scripts.js

```diff
@@ -7,10 +7,15 @@ if (h <= 0 ) {
 
 function drawTree(h) {
 	for (var n = 0; n <= h; n++) {
-		var star = '';
+		star = '';
```

Comment:
where is declaration for your variable `star`?  should be inside function :) last time this variable was declared globally, and this was wrong ;)